release/src/main/scripts/run_rc_validation.sh

```diff
@@ -15,15 +15,30 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
+# This script automates release candidate validation process.
+#
+# It reads configurations from script.config, checks environment settings and
+# runs a list of validation pipelines against multiple runners one after
+# another.
+#
+# NOTE:
+#   1. Please set all variables in script.config before running this script.
```

Comment:
this should be validated and we should probably still be asking the questions if the script has not been set up

Reply:
I do have a manual validation on line 79 where is the only interactive part (hopefully) in this script.